ui/src/pages/commands/commands.tsx

```diff
@@ -81,11 +84,18 @@ class Commands extends React.PureComponent<Props, State> {
   };
 
   render() {
+    let commands = this.props.devSpaceConfig.config.commands;
```

Comment:
I checked out your branch, fired it up and got the folllowing errors in the console, preventing the page to load

![ds-console-error](https://github.com/user-attachments/assets/5af83bdd-81ea-424b-bf59-f0c60cd96f0e)

Reply:
Sorry about that.
Trusting typescript is always dodgy.

Obviously the definitions in 
https://github.com/devspace-sh/devspace/blob/a633a2b35d4564b45c5fecfbb4646ccb331e4ae7/ui/src/contexts/withDevSpaceConfig/withDevSpaceConfig.tsx

are not proper.
Give it another try.